### ISSUE TITLE
New examples page proposal

### ISSUE THREAD

**Issue by kvolkovich-sc** [nikgraf/belle#337]

**Comment by nikgraf** [comment]
Sounds interesting! Do you use showroom in your company? I'm mostly concerned about long-term support.

**Comment by kvolkovich-sc** [comment]
Yes. We use showroom in our company about 4 months in 20+ repositories .
We very interested in further improvement and support.
